packages/typegen/src/generate/consts.ts

```diff
@@ -17,6 +18,21 @@ import { ignoreUnusedLookups } from './lookup.js';
 
 const generateForMetaTemplate = Handlebars.compile(readTemplate('consts'));
 
+function getDeprecationNotice (deprecationInfo: ItemDeprecationInfoV16, name: string): string {
```

Comment:
I think we can have just one single `getDeprecationNotice` by using Generics.

so:

```typescript
function getDeprecationNotice<T>(deprecationInfo: T, name: string)
```

Then we can call it by doing:

```typescript
getDeprecationNotice<ItemDeprecationInfoV16>(foo, bar);
```